src/main/resources/pdf-map.yaml

```diff
@@ -200,6 +209,33 @@ inputFields:
   homeExpenseWaterFrequency: expense_water_frequency
   homeExpenseWoodOrCoalFrequency: expense_wood_frequency
 
+  expenseSection8Housing: expense_section_8_bool
+  expenseHeatIncludedRent: expense_heat_included_rent_bool
```

Comment:
I'm noticing these don't show up automatically, but I suspect it's because they are lists in the data: 

```
"expenseSection8Housing[]": [
    "true"
  ],
  "expenseHeatIncludedRent[]": [
    "true"
  ],
```

Reply:
ah! This is the issue I was coming up on! I couldn't figure out why these checkmarks were not showing up